examples/03_connectivity/plot_sphere_based_connectome.py

```diff
@@ -449,4 +447,4 @@
 #
 #   * :ref:`sphx_glr_auto_examples_03_connectivity_plot_atlas_comparison.py`
 #
-#   * :ref:`sphx_glr_auto_examples_03_connectivity_plot_multi_subject_connectome.py`
+#   * :ref:`sphx_glr_auto_examples_03_connectivity_plot_multi_subject_connectome.py` # noqa
```

Comment:
The noqa here is necessary because the line is too long and I cannot think of a way to make it shorter.

I suppose this will show in the doc though.

Any alternative are welcome.